docs/docs/contributing/infrastructure.rst

```diff
@@ -0,0 +1,153 @@
+  `netcup.de`_. Costs exactly 1.44€ each year so quite sustainable, but makes
+  @ix5 a potential bus factor.
+* Website content is served by GitHub Pages. The `sphinx-doc`_-built site is
+  auto-deployed to `isso-comments.github.io`_ via a `GitHub Action`_ on every push
+  to the Isso main repository. Requires appropriate ``A`` and ``CNAME`` records
+  to GitHub's servers:
+
+  .. code-block:: console
+
+      $ dig www.isso-comments.de
+      www.isso-comments.de.	300	IN	CNAME	isso-comments.github.io.
+      isso-comments.github.io. 3600	IN	A	185.199.108.153
+      isso-comments.github.io. 3600	IN	A	185.199.109.153
+      isso-comments.github.io. 3600	IN	A	185.199.110.153
+      isso-comments.github.io. 3600	IN	A	185.199.111.153
+
+* The demo instance of the comment form (on the homepage) lives on `@ix5`_'s
+  server at `comments.isso-comments.de`_ and consists of the latest ``isso``
+  package from ``PyPI``, deployed via ``gunicorn``.
+  The ansible role to set this up is available at `ansible-role-isso`_. The
+  simple auto-reset feature to curtail vandalism is documented at
+  `isso-demo-config`_.
+
+.. _@posativ: https://github.com/posativ
+.. _posativ.org/isso: https://posativ.org/isso
+.. _isso-comments.de: https://isso-comments.de
+.. _@ix5: https://github.com/ix5
+.. _netcup.de: https://netcup.de
+.. _sphinx-doc: https://www.sphinx-doc.org/
+.. _isso-comments.github.io: https://github.com/isso-comments/isso-comments.github.io
+.. _GitHub Action: https://github.com/isso-comments/isso/blob/master/.github/workflows/build-upload-docs.yml
+.. _comments.isso-comments.de: https://comments.isso-comments.de
+.. _ansible-role-isso: https://git.ix5.org/felix/ansible-role-isso
+.. _isso-demo-config: https://github.com/isso-comments/isso-demo-config
+
+Development
+-----------
+
+Development happens under the `isso-comments GitHub organisation`_, with the
+main `isso-comments/isso`_ repository serving as the source for the Python
+parts, Javascript client, website and documentation, API documentation as well
+as CI/CD configuration.
+
+The ``master`` branch has branch protections set up, requiring an approving
+maintainer review before merging.
+
+.. _isso-comments GitHub organisation: https://github.com/isso-comments
+.. _isso-comments/isso: https://github.com/isso-comments/isso
+
+Packaging
+---------
+
+Isso is released "officially" to as an installable Python package on `PyPI`_
+and as a docker image. Other distributors may package releases of Isso natively
+for operating systems (e.g. for the Arch User repository or formerly Debian),
+but support for these releases should be given by the packager.
+
+.. _PyPI: https://pypi.org/project/isso/
+
+PyPI (Python Package Index)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Release rights for the `isso PyPI`_ project are held by @jelmer, @posativ, @blatinier and @ix5.
+
+Since `PGP support was dropped from PyPI`_ in 2023, PGP keys for signing the
+package are no longer relevant.
+
```

Comment:
This seems irrelevant here.

Reply:
My intention was to convey that there are no "release keys" _of significance to PyPI_ being held by anyone, but I agree with you, that's history and not currently relevant.